interpretation/ww_head.py

```diff
@@ -32,6 +33,38 @@ def process(message, isGameMaster=False, isAdmin=False):
     if is_command(message,['poll','testpoll']):
         return [Mailbox().new_poll(message.channel.id,'wolf',message.author.id,message.content.split(' ',1)[1])]
 
+    # =============================================================
+    #
+    #                         BOT COMMANDS
+    #
+    # =============================================================
+    if isPeasant == True:
+        if is_command(message,['warn'],False,unip):
+            # Warn the user that they've been active for a while.
+            answer = Mailbox().story("Hey there, <@{}>! You have been idling for about 48 hours now!\n".format(message.mentions[0].id))
+            return [answer.story_add("Please let us hear from you within 24 hours, or you will be disqualified for idling out.")]
+
+        if is_command(message,['idle'],False,unip):
+            # Kill the inactive user
+            answer = Mailbox().story("<@{}> has been killed due to inactivity.".format(message.mentions[0].id))
+            return [answer]
+
+        if is_command(message,['pay'],False,unip):
+            # Initiate the first round of starting the day.
+            return [Mailbox(True)]
+
+        if is_command(message,['day'],False,unip):
+            # Initiate the second round of starting the day.
+            pass
+
+        if is_command(message,['pight',False,unip]):
+            # Initiate the first round of starting the night.
+            return [Mailbox(True)]
+
+        if is_command(message,['night'],False,unip):
+            # Initiate the second round of starting the night.
+            pass
```

Comment:
are you supposed to `pass` here, or do you want to return a mailbox?

Reply:
There's a lot that needs to happen to initiate the night, but its functions is simply not written yet. We could return the todo() function, but given that this command is only used by bots, I supposed it wouldn't be necessary to use a placeholder.